src/main/java/com/sfuentes/date/DateProgram.java

```diff
@@ -0,0 +1,35 @@
+package com.sfuentes.date;
+
+import lombok.RequiredArgsConstructor;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Arrays;
+
+@RequiredArgsConstructor
+public class DateProgram {
+
+  private final String date;
+
+  public String convertDateToWords() {
+
+    var listOfStr = Arrays.asList(date.split("/"));
```

Comment:
Here you can use the array returned by method split, as far as I can see you are not using any List specific feature.

Reply:
Fixed it.